src/main/resources/messages/message_en_US.properties

```
@@ -53,7 +55,7 @@ msg.project.android.noticefile.required=Please upload Notice.html File.
 #msg.distribute.complete = Distribution has been completed.
 msg.distribute.reset = Distributed information has been initialized.
 #msg.distribute.confrim.overwritefile = A file with the same name already exists in the category.  Do you want to continue? <br />(If you change the file, it applies to all relevant models, regardless of release date.)
-msg.distribute.confrim.overwritefile=File(<b>{0}<b/>) with the same name already exists in the category. <span style="color:red;">Do you want to overwrite <b>{0}<b/> file?</span><br>(If you change the file, it applies to <span style="color:blue;">all relevant models that already distributed.</span>)
+msg.distribute.confrim.overwritefile = \ub3d9\uc77c\ud55c Category\uc5d0 \ub3d9\uc77c\ud55c \uc774\ub984\uc758 \ud30c\uc77c(<b>{0}<b/>)\uc774 \uc774\ubbf8 \uc874\uc7ac\ud569\ub2c8\ub2e4. <span style="color:red;"><b>{0}<b/> \ud30c\uc77c\uc744 \ub36e\uc5b4 \uc50c\uc6b0\uaca0\uc2b5\ub2c8\uae4c?</span><br>(\ub9cc\uc57d \ud574\ub2f9 \ud30c\uc77c\uc744 \ub36e\uc5b4\uc50c\uc6b0\uac8c \ub420 \uacbd\uc6b0, \ubcc0\uacbd \uc0ac\ud56d\uc740 <span style="color:blue;">\uc774\ubbf8 \ubc30\ud3ec\ub418\uc5c8\ub358 \ubaa8\ub378</span>\uc5d0\ub3c4 \uc801\uc6a9\ub429\ub2c8\ub2e4.)<br>File(<b>{0}<b/>) with the same name already exists in the category. <span style="color:red;">Do you want to overwrite <b>{0}<b/> file?</span><br>(If you change the file, it applies to <span style="color:blue;">all relevant models that already distributed.</span>)
```

유니코드 없이 아래와 같이 나와야합니다. 수정 부탁드려요.

```suggestion
msg.distribute.confrim.overwritefile=File(<b>{0}<b/>) with the same name already exists in the category. <span style="color:red;">Do you want to overwrite <b>{0}<b/> file?</span><br>(If you change the file, it applies to <span style="color:blue;">all relevant models that already distributed.</span>)
```